specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+            "$ref": "#/definitions/ComplianceReportItem"
+          },
+          "x-ms-identifiers": []
+        },
+        "compliancePdfReport": {
+          "description": "compliance pdf report",
+          "type": "object",
+          "readOnly": true,
+          "properties": {
+            "sasUri": {
+              "description": "uri of compliance pdf report",
+              "type": "string",
+              "readOnly": true,
+              "x-ms-secret": true
+            }
+          }
+        },
+        "complianceDetailedPdfReport": {
+          "description": "compliance detailed pdf report",
+          "type": "object",
+          "readOnly": true,
+          "properties": {
+            "sasUri": {
+              "description": "uri of compliance detailed pdf report",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "ComplianceReportItem": {
+      "type": "object",
+      "description": "Object that includes all the content for single compliance result.",
+      "properties": {
+        "categoryName": {
+          "type": "string",
+          "description": "The category name.",
+          "readOnly": true
+        },
+        "controlId": {
+          "type": "string",
+          "description": "The control Id.",
+          "readOnly": true
+        },
+        "controlName": {
+          "type": "string",
+          "description": "The control name.",
+          "readOnly": true
+        },
+        "controlType": {
+          "type": "string",
+          "description": "The control type.",
+          "$ref": "#/definitions/ControlType",
+          "readOnly": true
+        },
+        "complianceState": {
+          "type": "string",
+          "description": "The compliance result's status.",
+          "readOnly": true
+        },
+        "policyId": {
+          "type": "string",
+          "description": "The compliance result mapped policy Id.",
+          "readOnly": true
+        },
+        "policyDisplayName": {
+          "type": "string",
+          "description": "The policy's display name.",
+          "readOnly": true
+        },
+        "policyDescription": {
+          "type": "string",
+          "description": "The policy's detail description.",
+          "readOnly": true
+        },
+        "subscriptionId": {
+          "type": "string",
+          "description": "The compliance result mapped subscription Id.",
+          "readOnly": true
+        },
+        "resourceGroup": {
+          "type": "string",
+          "description": "The compliance result mapped resource group.",
+          "readOnly": true
+        },
+        "resourceType": {
+          "type": "string",
+          "description": "The compliance result mapped resource type.",
+          "readOnly": true
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "The compliance result mapped resource Id.",
```

Comment:
>       "description": "The compliance result mapped resource Id.", [](http://example.com/codeflow?start=3&length=66)

Should this be format arm-id? Probably doesn't matter much as long as its readonly.

Reply:
Yes, it is for the arm resource id, like /subscriptions/{subscriptionId}/resourceGroup/{rg_name}/{RP namespace}/.....

Reply:
Fixed, example added.
Thanks Tim~